patches/diffable-html+4.0.0.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/node_modules/diffable-html/dist/diffable-html.js b/node_modules/diffable-html/dist/diffable-html.js
+index 0d7d767..98fbad8 100644
+--- a/node_modules/diffable-html/dist/diffable-html.js
++++ b/node_modules/diffable-html/dist/diffable-html.js
+@@ -21,7 +21,7 @@ var voidElements = [
+   'track',
+   'wbr' ];
+ 
+-var format = function(html) {
++var format = function(html, { sortAttributes = undefined } = {}) {
```

Comment:
could you not open a PR to diffable html for this?

Reply:
It looks like he did but then went for the patch solution. https://github.com/rayrutjes/diffable-html/pull/39